lib/settings/pages/comment_appearance_settings_page.dart

```diff
@@ -29,6 +29,9 @@ class _CommentAppearanceSettingsPageState extends State<CommentAppearanceSetting
   /// When toggled on, comments will show a row of actions to perform
   bool showCommentButtonActions = false;
 
+  /// when toogled on, comments will show intsnace of origin
+  bool showOriginInstance = false;
```

Comment:
Just a couple typos in the comment!

Also, if we're switching to 'Show User Instance', maybe it would be best to rename all instances of `showOriginInstance` to `commentShowUserInstance` or something similar?